tools/releaseBuild/azureDevOps/templates/release-CreateGitHubDraft.yml

```diff
@@ -0,0 +1,22 @@
+steps:
+- download: none
+
+- template: release-SetReleaseTagAndContainerName.yml
+
+- pwsh: |
+    $azcopy = "C:\Program Files (x86)\Microsoft SDKs\Azure\AzCopy\AzCopy.exe"
```

Comment:
I think we should move this to macos and use brew to make sure the tools are there.  This just assumes.

Reply:
not blocking

Reply:
It works currently, i think we should keep it as is. The agent guarantees AzCopy to be present: https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md#cli-tools